### PR TITLE
Show the red omnisharp flame when there is std error in omnisharp

### PR DESCRIPTION
**Pull request by akshita31**

### CODE REVIEW

File: src/observers/OmnisharpStatusBarObserver.ts [hunk 1]
```diff
@@ -3,13 +3,14 @@
...
 
-import { OmnisharpServerOnServerError, BaseEvent, OmnisharpOnBeforeServerInstall, OmnisharpOnBeforeServerStart, OmnisharpServerOnStop, OmnisharpServerOnStart, DownloadStart, InstallationStart, DownloadProgress } from "../omnisharp/loggingEvents";
+import { OmnisharpServerOnServerError, BaseEvent, OmnisharpOnBeforeServerInstall, OmnisharpOnBeforeServerStart, OmnisharpServerOnStop, OmnisharpServerOnStart, DownloadStart, InstallationStart, DownloadProgress, OmnisharpServerOnStdErr } from "../omnisharp/loggingEvents";
 import { BaseStatusBarItemObserver } from './BaseStatusBarItemObserver';
 
 export class OmnisharpStatusBarObserver extends BaseStatusBarItemObserver {
     public post = (event: BaseEvent) => {
         switch (event.constructor.name) {
             case OmnisharpServerOnServerError.name:
+            case OmnisharpServerOnStdErr.name:
```
Comment (rchande) [inline src/observers/OmnisharpStatusBarObserver.ts]: Do we only get the OnStdErr event when starting omnisharp? The message on line 14 makes it sound like that.

Comment (akshita31) [reply src/observers/OmnisharpStatusBarObserver.ts]: It is produced when the standard error stream on the omnisharp process produces an error. Modified the case to show that message instead

File: src/observers/OmnisharpStatusBarObserver.ts [hunk 2]
```diff
@@ -12,6 +12,10 @@ export class OmnisharpStatusBarObserver extends BaseStatusBarItemObserver {
             case OmnisharpServerOnServerError.name:
                 this.SetAndShowStatusBar('$(flame)', 'o.showOutput', 'rgb(218,0,0)', 'Error starting OmniSharp');
                 break;
+            case OmnisharpServerOnStdErr.name:
+                let msg = (<OmnisharpServerOnStdErr>event).message;
+                this.SetAndShowStatusBar('$(flame)', 'o.showOutput', 'rgb(218,0,0)', `OmniSharp process errored:${msg}`);
```
Comment (rchande) [inline src/observers/OmnisharpStatusBarObserver.ts]: Can you extract out constants for the colors and call them names like "red" and "green"?

File: test/unitTests/logging/OmnisharpStatusBarObserver.test.ts [hunk 3]
```diff
@@ -27,17 +27,31 @@ suite('OmnisharpStatusBarObserver', () => {
...
-    test('OnServerError: Status bar is shown with the error text', () => {
-        let event = new OmnisharpServerOnServerError("someError");
+    [
+        new OmnisharpServerOnServerError("someError"),
+    ].forEach((event: BaseEvent) => {
+        test(`${event.constructor.name}: Status bar is shown with the error text`, () => {
+            observer.post(event);
+            expect(showCalled).to.be.true;
+            expect(statusBarItem.text).to.equal(`$(flame)`);
+            expect(statusBarItem.command).to.equal('o.showOutput');
```
Comment (rchande) [inline test/unitTests/logging/OmnisharpStatusBarObserver.test.ts]: Verify color?

**Pull request merged**